config/core/build-configs.yaml

```
@@ -384,6 +423,10 @@ build_environments:
     arch_params:
       <<: *clang_12_arch_params
 
+  rustc-1.62:
+    cc: clang
+    cc_version: 15
```

```suggestion
  rustc-1.62:
    cc: clang
    cc_version: 15
    arch_params:
      x86_64: 
        <<: *x86_64_params
        name:
```
I think we can explicitly set an empty architecture name here. Otherwise jenkins build job will be looking for an `rustc-1.62-x86_64` docker image to get a proper toolchain. As long as we have just one docker image template for this config, I think we don't need to suffix it with an arch. 
Having said that, if you think it's worth adding an arch suffix to the docker image then `rustc-1.62.jinja2` template should be renamed to `rustc-1.62-x86_64.jinja2`.